.github/workflows/python-tests.yml

```diff
@@ -6,6 +6,8 @@ on: [push, pull_request]
 jobs:
   Get-CI-Image-Tag:
     uses: ./.github/workflows/get-ci-image-tag.yml
+    with:
+      product: opensearch
```

Comment:
We add this to opt out the OSD python tests?

Reply:
This has nothing to do with OSD python test it is just more consistent to find the AL2 image we are looking for.